### PR TITLE
Added docs for CCI stochastic.

### PR DESCRIPTION
**Pull request by quantum8**
Added docs for CCI Divergence and cleaned up divergences docs files.

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json [hunk 1]
```diff
@@ -1,74 +0,0 @@
-{
-    "topic": "Divergences Data Mine",
-    "pageNumber": 3,
-    "type": "Commodity Channel Index Indicator - CCI",
-    "definition": {
-        "text": "This indicator produces the Commodity Channel Index oscillator, as originally published by Donald Lambert in 1980."
-    },
-    "paragraphs": [
-        {
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json:9]: Will there be replacement for this page?

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json [hunk 2]
```diff
@@ -4,128 +4,394 @@
...
+                    "language": "TR",
+                    "text": "Farklılıklar bulucusunun uzunluğu, pivot uzunluğu ve ayrıca güçlü ve zayıf sapmalar için eşik, Javascript Kodunun ( Javascript Code ) 'RSI Sapma'Prosedür Döngüsü altındaki Veri Oluşturma Prosedürü ( Data Building Procedure ) -> Prosedür Döngüsü ( Procedure Loop ) altında bulunması ve açılmasıyla değiştirilebilir.",
+                    "updated": 1647002818095
+                }
+            ]
+        },
         {
             "style": "Png",
             "text": "PNGs/Foundations/Docs/indicators/Divergences-RSI-params-1.png"
-        },
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: Is this part still needed when down below there comes:
style
text 
translations?

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json (hunk 1 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json:9]: I'm trying to consolidate the files as there's been some duplicates added here from SA when the translation was added, which is confusing a few things now!

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json (hunk 2 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: not sure what you mean by this?

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json (hunk 1 above)
Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json:9]: So you are sure this file should get deleted?

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json (hunk 2 above)
Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: you delete a small part here. Above there is already:
style
text
 that wont get deleted. but down below the part where you delete something there got added :
style
text
translations
 and Iam unsure if the first part is a duplicate or if this is just the syntax.

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json (hunk 1 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-003-commodity-channel-index-indicator.json:9]: yeah I've replaced it with the file: divergences-data-mine-003-commodity-channel-index-indicator-cci.json

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json (hunk 2 above)
Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: hmm strange, that file was duplicated altogether so i removed the incorrect one and added one with the filename divergences-data-mine-002-rsi-divergence.json. Maybe git is being weird about the filename case on the new file.

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: So now everything should be correct?

Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-002-RSI-Divergence.json:69]: yeah should be good there now I think!

**Review by gitforkgit — APPROVED** [review]
I merge it but please do me the favor and look in retrospect if problems have occurred. :+1:

**Comment by quantum8** [comment]
yes will do @gitforkgit !

**Pull request merged**